docs/data-cookbooks/sips-workflow.md

```diff
@@ -0,0 +1,133 @@
+  	    },
+  	    {
+  	        "bucket": "protected-2",
+  	        "regex": "^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006\\.[\\d]{13}\\.cmr\\.xml$",
+  	        "sampleFileName": "MOD09GQ.A2017025.h21v00.006.2017034065104.cmr.xml"
+  	    },
+  	    {
+  	        "bucket": "public",
+  	        "regex": "^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006\\.[\\d]{13}_ndvi\\.jpg$",
+  	        "sampleFileName": "MOD09GQ.A2017025.h21v00.006.2017034065104_ndvi.jpg"
+  	    }
+    ],
+    "duplicateHandling": "replace",
+    "updatedAt": 1533313794693,
+    "url_path": "{cmrMetadata.Granule.Collection.ShortName}/{substring(file.name, 0, 3)}",
+    "timestamp": 1533313798525,
+    "stats": {
+        "running": 0,
+        "completed": 0,
+        "failed": 2,
+        "total": 2
+    }
+}
+```
+
+#### Rule
+
+Finally, let's create a [rule](./setup.md#rules). In this example we're just going to create a `onetime` throw-away rule that will be easy to test with. This rule will kick off the `DiscoverAndParsePdrs` workflow, which is the beginning of a Cumulus SIPS workflow.
+
+```
+{
+    "name": "s3_provider_rule",
+    "workflow": "DiscoverAndQueuePdrs",
+    "provider": "s3_provider",
+    "collection": {
+        "name": "MOD09GQ",
+        "version": "006"
+    },
+    "rule": {
+        "type": "onetime"
+    },
+    "state": "ENABLED",
+    "tags": [
+        "test"
+    ]
+}
+```
+
+**Note:** A list of configured workflows exists under the "Workflows" in the navigation bar on the Cumulus dashboard. Additionally, one can find a list of executions and their respective status in the "Executions" tab in the navigation bar.
+
+
+## DiscoverAndQueuePdrs Workflow
+
+This workflow will (as the name might suggest) discover pdrs and queue them to be processed. Duplicate PDRs will be dealt with according to the configured duplicate handling setting in the collection.
+
+1. DiscoverPdrs - [npm package](https://www.npmjs.com/package/@cumulus/discover-pdrs), [source](https://github.com/nasa/cumulus/tree/master/tasks/discover-pdrs)
+2. QueuePdrs - [npm package](https://www.npmjs.com/package/@cumulus/queue-pdrs), [source](https://github.com/nasa/cumulus/tree/master/tasks/queue-pdrs)
+
+![](../images/sips-discover-and-queue-pdrs-execution.png)
+
+_Example configuration for this workflow can be found in the `DiscoverAndQueuePdrs` object defined in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/workflows.yml)_
+
+
+## ParsePdr Workflow
+
+The ParsePdr workflow will parse a pdr, queue the specified granules (duplicates are handled according to the duplicate handling setting) and periodically check the status of those queued granules. This workflow will not succeed until all the granules included in the pdr are successfully ingested. If one of those fails, the ParsePdr worklfow will fail. **NOTE** that ParsePdr may spin up multiple IngestGranule workflows in parallel, depending on the granules included in the pdr.
```

Comment:
PDR

Reply:
oh boy, I missed a bunch of those. Good call